cmd/kyma/apply/function/function.go

```diff
@@ -0,0 +1,230 @@
+	c := command{
+		opts:    o,
+		Command: cli.Command{Options: o.Options},
+	}
+	cmd := &cobra.Command{
+		Use:   "function",
+		Short: "Applies local resources for your Function to the Kyma cluster.",
+		Long: `Use this command to apply the local sources of your Function's code and dependencies to the Kyma cluster. 
+Use the flags to specify the desired location for the source files or run the command to validate and print the output resources.`,
+		RunE: func(cmd *cobra.Command, args []string) error {
+			return c.Run()
+		},
+	}
+
+	cmd.Flags().StringVarP(&o.Filename, "filename", "f", "", `Full path to the config file.`)
+	cmd.Flags().BoolVar(&o.DryRun, "dry-run", false, `Validated list of objects to be created from sources.`)
+	cmd.Flags().Var(&o.OnError, "onerror", `Flag used to define the Kyma CLI's reaction to an error when applying resources to the cluster. Use one of these options: 
+- nothing
+- purge`)
+	cmd.Flags().VarP(&o.Output, "output", "o", `Flag used to define the command output format. Use one of these options:
+- text
+- json
+- yaml
+- none`)
+
+	return cmd
+}
+
+func (c *command) Run() error {
+	if c.opts.Filename == "" {
+		c.opts.Filename = defaultFilename()
+	}
+
+	file, err := os.Open(c.opts.Filename)
+	if err != nil {
+		return err
+	}
+
+	// Load project configuration
+	var configuration workspace.Cfg
+	if err := yaml.NewDecoder(file).Decode(&configuration); err != nil {
+		return errors.Wrap(err, "Could not decode the configuration file")
+	}
+
+	if c.K8s, err = kube.NewFromConfig("", c.KubeconfigPath); err != nil {
+		return errors.Wrap(err, "Could not initialize the Kubernetes client. Make sure your kubeconfig is valid")
+	}
+	client := c.K8s.Dynamic()
+
+	function, err := resources.NewFunction(configuration)
+	if err != nil {
+		return err
+	}
+
+	triggers, err := resources.NewTriggers(configuration)
+	if err != nil {
+		return err
+	}
+
+	operators := map[operator.Operator][]operator.Operator{
+		operator.NewGenericOperator(client.Resource(operator.GVKFunction).Namespace(configuration.Namespace), function): {
+			operator.NewTriggersOperator(client.Resource(operator.GVKTriggers).Namespace(configuration.Namespace), triggers...),
+		},
+	}
+
+	if configuration.Source.Type == workspace.SourceTypeGit {
+		gitRepository, err := resources.NewPublicGitRepository(configuration)
+		if err != nil {
+			return errors.Wrap(err, "Unable to read the Git repository from the provided configuration")
+		}
+		gitOperator := operator.NewGenericOperator(client.Resource(operator.GVRGitRepository).Namespace(configuration.Namespace), gitRepository)
+		operators[gitOperator] = nil
+	}
+
+	mgr := manager.NewManager(operators)
+	options := manager.Options{
+		Callbacks:          callbacks(c),
+		OnError:            chooseOnError(c.opts.OnError),
+		DryRun:             c.opts.DryRun,
+		SetOwnerReferences: true, // add one more flag to set it to false or add: c.opts.Output != YAMLOutput && c.opts.Output != JSONOutput
```

Comment:
Was this supposed to be added in this PR or will there be another issue for it?

Reply:
After merging this PR I will log an issue for this